### ISSUE TITLE
Inconsistent handbook/docs width

### ISSUE THREAD

**Issue by joepavitt** [FlowFuse/website#2259]
### Current Behavior

There is a windows between 1000ish and 1400ish pixels where a max width adds margin either side:

<img width="1434" alt="Screenshot 2024-06-28 at 12 23 33" src="https://github.com/FlowFuse/website/assets/99246719/bcad44b7-3ea6-4739-930a-bcb9199d0795">

On larger screens (and smaller), it's then shown as full width:

<img width="1714" alt="Screenshot 2024-06-28 at 12 23 41" src="https://github.com/FlowFuse/website/assets/99246719/efa52c1f-001c-46b3-a41c-0cbf1ea8ae8b">

### Expected Behavior

I think the full width on the wider screens is a good move, but it'd be better to have wider gutters for the left-right content over 1400ish pixels as they're very narrow given thew width of the content.

### Environment

- Date:
- Platform/OS:
- Browser:

### Have you provided an initial effort estimate for this issue?

I have provided an initial effort estimate

**Comment by Yndira-E** [comment]
Hi @joepavitt, the max-width is 1920px. So the margins come back at larger screens.

By wider gutters you mean the right-left margins of the middle column? or is it the width of the right-left columns what you're suggesting should be increased?

**Comment by joepavitt** [comment]
> the max-width is 1920px.

My first screenshot above shows otherwise:

<img width="187" alt="Screenshot 2024-06-28 at 12 38 21" src="https://github.com/FlowFuse/website/assets/99246719/9aed9482-cae8-4891-9563-7815a16ee44e">

Anything between `1024` and `1534` will render with margins

**Comment by joepavitt** [comment]
> is it the width of the right-left columns what you're suggesting should be increased?

This, on larger screens, having them 350px, or something like that will improve the sizing I think

**Comment by Yndira-E** [comment]
> My first screenshot above shows otherwise:

It has two breakpoints, depending on your screen size you'll see a different max-width.
<img width="180" alt="Screenshot 2024-06-28 at 13 43 14" src="https://github.com/FlowFuse/website/assets/129537638/264d220a-011f-4e77-a662-1153f102fde0">

The logo moves to align with the wider content, so for medium-sized screens, I set a max-width to keep the logo at a sensible distance from pages that use a different layout. For large screens, I set a larger max-width and accepted that the logo would be farther away.
